package/tests/test_models.py

```diff
@@ -7,6 +7,23 @@ class VersionTests(TestCase):
     def setUp(self):
         data.load()
 
+    def test_ranking(self):
```

Comment:
@RamezIssac can we have more tests for a package that is last updated like 4 months ago, 9 months ago, 13 months ago 2.5 years ago?

I would suggest also add a test case to make sure your logic is working if the repo is archived or deleted from the GitHub